spyrmsd/molecule.py

```diff
@@ -45,17 +45,62 @@ def __init__(
         self.atomicnums = atomicnums
         self.coordinates = coordinates
 
-        self.stripped: bool = np.all(atomicnums != 1)
+        self.stripped: bool = bool(np.all(atomicnums != 1))
```

Comment:
NumPy booleans have type `bool_`, therefore an explicit conversion is needed for `mypy`.